### DISCUSSION TITLE
code analysis c++-20 (and beyond)

### DISCUSSION THREAD

**Discussion by hros** [microsoft/vscode-cpptools#9108] [Q&A] [answered]
how can I set the integrated clang-tidy to recognize c++-20?
I tried adding "--std=c++20" to `Clang Tidy: Args` but that did not make any change (code that compiles correctly without any warnings shows many linter errors)

**Comment by sean-mcmanus** [comment, marked as answer]
Our clang-tidy feature currently gets most of its configuration from IntelliSense, so you need to set the cppStandard setting in c_cpp_properties.json or C_Cpp.default.cppStandard.
